sections/shared/Layout/AppLayout/Header/BalanceActions/BalanceActionsContainer.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components';
+import { Synths } from '@synthetixio/contracts-interface';
+import useSynthetixQueries from '@synthetixio/queries';
+import CurrencyIcon from 'components/Currency/CurrencyIcon';
+import useGetFuturesMarkets from 'queries/futures/useGetFuturesMarkets';
+import useGetFuturesPositionForMarkets from 'queries/futures/useGetFuturesPositionForMarkets';
+import React, { FC, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { walletAddressState } from 'store/wallet';
+import { zeroBN } from 'utils/formatters/number';
+import UniswapModal from 'sections/shared/modals/UniswapModal';
+
+import Button from 'components/Button';
+import { useTranslation } from 'react-i18next';
+import BalanceActions from './BalanceActions';
+
+const BalanceActionsContainer: FC = () => {
+	const [uniswapWidgetOpened, setUniswapWidgetOpened] = useState<boolean>(false);
+	const { t } = useTranslation();
+
+	const futuresMarketsQuery = useGetFuturesMarkets();
+	const futuresMarkets = futuresMarketsQuery?.data ?? [];
+	const futuresPositionQuery = useGetFuturesPositionForMarkets(
+		futuresMarkets.map(({ asset }) => asset)
+	);
+	const futuresPositions = futuresPositionQuery?.data ?? [];
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+
+	const sUSDBalance = synthsBalancesQuery?.data?.balancesMap?.[Synths.sUSD]?.balance ?? zeroBN;
+
+	if (!sUSDBalance) {
+		return null;
+	}
+
+	return (
+		<Container>
+			{sUSDBalance === zeroBN ? (
+				<StyledButton onClick={() => setUniswapWidgetOpened(true)}>
+					<StyledCurrencyIcon currencyKey={Synths.sUSD} />
+					{t('header.balance.get-susd')}
```

Comment:
Can you make sure `text-transform: capitalize` is turned off here so it does not capitalize the `s` in `sUSD`?

<img width="151" alt="Screen Shot 2022-04-07 at 16 07 30" src="https://user-images.githubusercontent.com/548702/162278279-ce25d3c6-9f23-49f3-8b84-fa5798705752.png">